apps/docs/src/pages/components/app-switcher.mdx

```diff
@@ -0,0 +1,129 @@
+import { Callout } from "nextra/components";
+import {
+  appSwitcherClasses,
+  HvAppSwitcher,
+} from "@hitachivantara/uikit-react-core";
+import { Code, LeftAlign } from "@hitachivantara/uikit-react-icons";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData(
+    "AppSwitcher",
+    "core",
+    appSwitcherClasses,
+    ["Action"],
+  );
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+<Playground
+  Component={HvAppSwitcher}
+  componentName="HvAppSwitcher"
+  controls={{
+    layout: {
+      defaultValue: "single",
+    },
+    title: {
+      defaultValue: "My Apps",
+    },
+    header: {
+      defaultValue: "Header",
+    },
+    footer: {
+      defaultValue: "Footer",
+    },
+  }}
+  componentProps={{
+    applications: [
+      {
+        name: "UI Kit Storybook",
+        color: "#FF4785",
+        iconUrl:
+          "https://pbs.twimg.com/profile_images/1100804485616566273/sOct-Txm_400x400.png",
+        description: "This is the Storybook for the UI Kit project",
+        url: "https://lumada-design.github.io/uikit/master/",
+        target: "_top",
+      },
+      {
+        name: "UI Kit repository (New Tab)",
+        description: "This is the UI Kit repository on Github",
+        url: "https://github.com/lumada-design/hv-uikit-react/",
+        target: "_blank",
+      },
+      {
+        name: "App with a bigger name than the other just to showcase the truncation on the AppSwitcher",
+        color: "primary",
+        description: "App 1 description",
+      },
+      {
+        name: "No Icon App",
+        color: "brand",
+        description: "This is a disabled App without an icon",
+        disabled: true,
+      },
+      {
+        name: "No Description App",
+        color: "brand",
+      },
+    ],
+  }}
+/>
+
+### Many entries
+
+When there are many entries, the AppSwitcher will scroll vertically. Use the `isActionSelectedCallback` prop to highlight the selected application.
+
+```tsx live
+export default function Demo() {
+  return (
+    <div className="flex h-360px">
+      <HvAppSwitcher
+        layout="dual"
+        title="Big list of applications"
+        applications={getDummyApplicationsList()}
+        isActionSelectedCallback={handlesActionSelectedCallback}
+      />
+    </div>
+  );
+}
+
+const handlesActionSelectedCallback: HvAppSwitcherProps["isActionSelectedCallback"] =
+  (application) => {
+    if (typeof window !== "undefined") {
+      return window.location.href.startsWith(application.url || "");
+    }
+    return false;
+  };
```

Comment:
I don't think it's worth it to keep the `window` checks. Most users won't be using SSR, and even those who do, this component will be hidden inside a client-side pop-up.
Also, I believe we don't want an empty/nullish URL to match as `true`

Or we could just remove it all together, It won't work in this page with the current links anyways
```suggestion
        isActionSelectedCallback={(app) => app.url && location.href.startsWith(app.url)}
      />
    </div>
  );
}
```